### ISSUE TITLE
Jas does not announce column or row headers in grids when virtual mode is toggled off with Jaws Key + z

### ISSUE THREAD

**Issue by Wildebrew** [FreedomScientific/VFO-standards-support#929]
When users use the keyboard shortcut Jawskey-z to toggle the virtual cursor off in a grid, column or row headers are no longer announced as they navigate.

To test this:
1. Go to [This ARIA Authoring Practices grid example](https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/)
2. Press JawsKey + z to turn virtual cursor off
3. Navigate to the grid and then user arrow keys to navigate between cells

Notice how the column headers, "type, description etc." are not announced at all.
If, instead, you toggle virtual cursor on, set forms mode to "auto", and repeat the navigation steps described above, the column headers are announced as you navigate the grid.

Why is this important?
Users who prefer to manually toggle forms mode on and off have trouble activating forms mode in a grid.
Also some users prefer to turn virtual cursor off when navigating a grid with keyboard.

Expectation:
Even if you turn off the virtual cursor you should still hear the programmatically associated info when you navigate with the keyboard, including associated labels, group labels and, in this case, column or row headers in a grid.

**Comment by JeremiahRogersVispero** [comment]
This works correctly for me in JAWS 2026.2512.50.
